internal/webapp/views/layout/base_templ.go

```diff
@@ -0,0 +1,204 @@
+// Code generated by templ - DO NOT EDIT.
+
+// templ: version: 0.2.476
+package layout
+
+//lint:file-ignore SA4006 This context is only used if a nested component is present.
+
+import "github.com/a-h/templ"
+import "context"
+import "io"
+import "bytes"
+
+func Base(title string) templ.Component {
```

Comment:
Don't change this part please, passing just a title as the parameter is not a good practice. I will refactor this part and by using custom types make this template more dynamic.